cmd/kubelet/app/options/globalflags.go

```diff
@@ -105,6 +105,7 @@ func addGlogFlags(fs *pflag.FlagSet) {
 	register(global, local, "log_backtrace_at")
 	register(global, local, "log_dir")
 	register(global, local, "log_file")
+	register(global, local, "log_file_max_size")
```

Comment:
is it our intent to keep adding global flags like this? I expected us to manage flags in a better way going forward.

Reply:
Honestly I'm not sure. I updated the code where the codebase is referenced, I can take this one out I think, but the other change in components-base is required otherwise some tests fail.

Reply:
To be clear, the issue is with the change made in klog, but this change here is a good example of the further pollution of the global flag space it is causing. Before we pull it into kubernetes, we should resolve the future direction of klog flag handling.

Reply:
cc @dims

Reply:
@liggitt please see https://github.com/kubernetes/klog/pull/56 and https://github.com/kubernetes/klog/issues/55 we can continue the conversation there